server/spec/cert_revocation_spec.rb

```diff
@@ -84,7 +84,7 @@
     new_time = File.mtime("/var/lib/candlepin/candlepin-crl.crl")
     new_time.should_not == old_time
     crl = OpenSSL::X509::CRL.new File.read "/var/lib/candlepin/candlepin-crl.crl"
-    crl.revoked.map { |i| i.serial }.should include(serial)
+    expect(crl.revoked.map { |i| i.serial }).to include(serial.to_i)
```

Comment:
To be consistent with the core of this change, we should cast the CRL serials to a string, rather than the serial received to an integer/long.

Reply:
Done.